packages/core/src/shader/ShaderProgram.ts

```diff
@@ -262,10 +262,12 @@ export class ShaderProgram {
       return null;
     }
 
-    if (!gl.getProgramParameter(program, gl.LINK_STATUS)) {
-      Logger.error("Could not link WebGL program. \n" + gl.getProgramInfoLog(program));
-      gl.deleteProgram(program);
-      return null;
+    if (Logger.isEnabled) {
```

Comment:
`Logger.isEnabled && !gl.getProgramParameter(program, gl.LINK_STATUS)` is better

Reply:
> `Logger.isEnabled && !gl.getProgramParameter(program, gl.LINK_STATUS)` is better

Done